schemas/put-artifact-request.yml

```diff
@@ -0,0 +1,15 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Put Artifact Request"
```

Comment:
please name it `Complete Artifact Request` in title and filename. It related to the `completeArtifact` signature...